LUKeychainAccess/NSKeyedArchiver+Additions.m

```diff
@@ -2,7 +2,9 @@
 
 @implementation NSKeyedArchiver (Additions)
 
-+ (NSData *)lu_archivedDataWithRootObject:(nonnull id)object {
++ (NSData *)lu_archivedDataWithRootObject:(nullable id)object {
+  if (!object) return nil;
```

Comment:
So his was the main issue causing the `root` error warnings. For some reason  even if the object was nil `result` would still have a value of empty info